docs/examples/others/test_drag.html

```diff
@@ -0,0 +1,238 @@
+   var heatmapTrack = 
+    {
+        "server": "http://test1.resgen.io/api/v1",
+        "tilesetUid": "N4h3G6FjSOKs5GOIPANVEA",
+        "datatype": "matrix",
+    } 
+
+    console.log('heatmapDragStart(ev)');
+    window.hgApi.showAvailableTrackPositions(heatmapTrack);
+    //ev.dataTransfer.setData("text/json", JSON.stringify(heatmapTrack));
+}
+
+ var testViewConfig =
+{
+  "editable": true,
+  "trackSourceServers": [
+    "/api/v1",
+    "http://higlass.io/api/v1"
+  ],
+  "exportViewUrl": "/api/v1/viewconfs",
+  "views": [
+    {
+      "uid": "a",
+      "initialXDomain": [
+        1502666124.488715,
+        1511074199.6112716
+      ],
+      "initialYDomain": [
+        1505212490.8607008,
+        1510254940.471237
+      ],
+      "tracks": {
+        "top": [],
+        "left": [],
+        "center": [
+
+        ],
+        "bottom": [],
+        "right": []
+      },
+      "layout": {
+        "w": 12,
+        "h": 6,
+        "x": 0,
+        "y": 0,
+        "i": "a",
+        "moved": false,
+        "static": false
+      }
+    }
+  ],
+  "zoomLocks": {
+    "locksByViewUid": {},
+    "locksDict": {}
+  },
+  "locationLocks": {
+    "locksByViewUid": {},
+    "locksDict": {}
+  },
+  "valueScaleLocks": {
+    "locksByViewUid": {},
+    "locksDict": {}
+  }
+}
+ ;
+    function changeOverlay() {
+
+        hgcontainer = document.createElement('div');
+        hgcontainer.style.width = '100%';
+        // this.hgcontainer.style.margin = '1em';
+        // this.hgcontainer.style.boxSizing = 'border-box';
+        // this.hgcontainer.style.lineHeight = '48px';
+
+        hgdisplay = document.createElement('div');
+        hgdisplay.style.height = '600px';
+        hgdisplay.style.width = '100%';
+        hgdisplay.style.border = '1px solid black';
+        hgdisplay.style.borderRadius = '1%';
+        hgdisplay.style.borderColor = '#dddddd';
+
+        hgcontainer.appendChild(hgdisplay);
+
+        window.hgApi = hglib.viewer(
+                        hgdisplay,
+                        testViewConfig,
+                        { bounded: true }
+                    );
+
+
+      var el = document.getElementById('placeholder');
+      el.appendChild(hgcontainer);
+    }
+
+    changeOverlay();
+
+    // setTimeout(function () {
+    //   // Go to PTEN
+    //   window.hgApi.goTo(
+    //     testViewConfig.views[0].uid,
+    //     'chr10',
+    //     89596071,
+    //     89758810,
+    //     'chr10',
+    //     89596071,
+    //     89758810,
+    //     true,
+    //     10000
+    //   );
+    // }, 3000);
+
+    // setTimeout(function () {
+    //   // Go to Chr1 without animation
+    //   window.hgApi.goTo(
+    //     testViewConfig.views[0].uid,
+    //     'chr1',
+    //     0,
+    //     89758810,
+    //     'chr1',
+    //     0,
+    //     89758810
+    //   );
+    // }, 11000);
+
+
+</script>
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
Can we remove the GA code from examples?

Reply:
Yup, done!